components/formats-gpl/src/loci/formats/in/MetamorphReader.java

```diff
@@ -1030,6 +1030,9 @@ protected void initStandardMetadata() throws FormatException, IOException {
       if (uic4tagEntry != null) {
         mmPlanes = uic4tagEntry.getValueCount();
       }
+      if (mmPlanes == 0) {
+        mmPlanes = ifds.size();
```

Comment:
Is it worth adding a comment explaining under what circumstances this block should be executed?